HackerNewsReader/Classes/AppDelegate.m

```diff
@@ -36,6 +38,13 @@ - (void)setupAppirater {
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     [self setupAppirater];
 
+    // Basic integration with Sift.
+    Sift *sift = Sift.sharedInstance;
+    sift.accountId = @"<put your account id here>";
+    sift.beaconKey = @"<put your Javascript snippet key here>";
+    sift.userId = @"<put user id here>";
```

Comment:
And actually, if the point of this is to include it as an example in the public repo, then I think this is maybe a bad example -- we'd never want an actual app to hard-wire in a userid.